data/model-db/generate_lookup_tables.R

```diff
@@ -0,0 +1,422 @@
+# Script to generate lookup tables for db
+# This only needs to be run once unless there are modifications
```

Comment:
I think we need to update to add RBDD sites now that that is in catch and trap tables. Otherwise was working fine for me.

Reply:
All of the red bluff sites are in the trap_location lookup table. Were you getting errors related to that?